app/core.py

```
@@ -1,107 +1,11 @@
-            }
-        except httpx.RequestError as exc:
-            raise HTTPException(
-                status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
-                detail=f"An error occurred while sending the webhook to {exc.request.url!r}: {exc}"
-            )
-        except Exception as exc:
-            raise HTTPException(
-                status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
-                detail=f"An unexpected error occurred: {str(exc)}"
-            )
+from typing import Any, Dict, List, Union
```

Ruff (0.11.9)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)